spec/toggle-quotes-spec.js

```diff
@@ -184,6 +184,31 @@ describe('ToggleQuotes', () => {
         })
       })
     })
+
+    describe('when the string contains escaped characters', () => {
+      it('does not remove them if they are at the end of the line', () => {
```

Comment:
Can you help me understand the significant of escaped characters at the end of the string? Is there something about escaped characters at the end of the string that needs to be handled differently than escaped characters elsewhere in the string?

Reply:
It's been a while, not sure why I did that. It may be related to how it didn't use to count escapes, so it would see `\'` in `'\\'`

Reply:
But I've removed it because we have a spec for counting escapes anyway now